presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -122,6 +122,8 @@
     private int fileSystemMaxCacheSize = 1000;
 
     private boolean writesToNonManagedTablesEnabled;
+    private boolean nonManagedTableCreatesEnabled = true;
```

Comment:
`createsOfNonManagedTablesEnabled`? that way it is more coherent

Reply:
I can change that if you'd like.